open-hackathon-server/src/hackathon/hack/hackathon_manager.py

```diff
@@ -307,6 +307,20 @@ def upload_files(self):
 
         return {"files": images}
 
+    def get_userlike_all_hackathon(self, user_id):
```

Comment:
What's the purpose of this function? Please rename the method so we can infer its purpose from function name